docs/en/workflow/testing-with-mocks.md

```diff
@@ -14,9 +16,9 @@ Suppose we have a component like this:
 
 <script>
 // this dependency needs to be mocked
-import SomeService from '../service'
+const SomeService = require('../service')
 
-export default {
+module.exports = {
   data () {
```

Comment:
In ES5, I think you should be change to `data: function () {`.  :)

Reply:
Actually it just disables `esModule` option, not the whole ES6+ syntax. So that's OK imo :-)

Reply:
okay! :)